### ISSUE TITLE
Empty row in Cube Creator returns "null" value in Visualize

### ISSUE THREAD

**Issue by tboeni** [visualize-admin/visualization-tool#1110]
When displaying the [Quality of Bathing Water Cube](https://int.visualize.admin.ch/create/veyvM2A59KJ0?dataSource=Int) on INT the column  **Upper LOD** is supposed to be empty. This happens both in preview mode as in the table view
I gave it the same property as the Lower LOD column in Cube Creator which is "integer" ([Link](https://int.cube-creator.lindas.admin.ch/app/cube-projects/cube-project!!ubd01041-qualitat-der-badegewasser-schweiz-vem16vxjrk8/csv-mapping))
However, Visualize displays the **Upper LOD** not as an empty column but with the "null" value. This is an issue in terms of consitency and also because null could be interpreted as "zero" in German.
![LOD null](https://github.com/visualize-admin/visualization-tool/assets/111350367/f1973838-1e75-4027-a266-1c48e20cff6c)

**Comment by bprusinowski** [comment]
Hi @tboeni, I investigated the issue and it seems there are some issues with the data:
- the `Upper LOD` dimension is not marked as a numerical dimension, and we are treating it as a "namedNodeDimension". This means that we create a mapping of value – label to get the labels, and another problem appears here: the missing values are not marked with `https://cube.link/Undefined` value, but rather are simply empty (`null`). This means that we are omitting them from such mapping and simply convert actual value to string, which results in a "null" visible in the table. I assume there is a way to mark such values with `https://cube.link/Undefined` in the Cube Creator, because we have such logic to handle empty values since long time,
- the `Lower LOD` dimension also misses the `https://cube.link/Undefined` values, but it's marked as numerical dimension – in this case again we check is the value is equal to `https://cube.link/Undefined` and if it's not (and this is not the case, as it's `null`), we convert it to an empty string.

It seems that handling of undefined values is described in the: [Cube Schema documentation](https://cube.link/#null-empty-values), so I would advise to try to adjust the dimensions in a recommended way (change the Upper LOD data type to numerical and replace `nulls` with `https://cube.link/Undefined`) and then the values should be displayed correctly in Visualize.

Technically, we also could adjust the logic to handle `nulls` together with `https://cube.link/Undefined`, but I think it makes more sense to follow the guidelines here.

Let me know if everything is clear and it's possible to update the cube with the new empty values 👌

**Comment by tboeni** [comment]
Hi @bprusinowski  
Thanks for the answer!

After checking im my Cube Creator project I found no differences in "values" that I had given the two different rows. 
And as far as my checking went I did not find an option to mark rows / values as `cube:undefined`. I am not quite sure but from what you are telling me the source of this display issue comes from the way the data / values is handled in LINDAS? 
If so I might have to consult Zazuko on the matter 

Thanks already!
